repo-agent/issue-sandbox/issue-sandbox-rgd.yaml

```diff
@@ -95,6 +95,8 @@ spec:
                   image: ko://repo-agent/images/issue-sandbox
                   env:
                     # URL to the repository where the .devcontainer folder we want to load is located
+                    - name: AGENT_NAME
```

Comment:
The `AGENT_NAME` environment variable is defined here but does not seem to be used in the `issue-sandbox/main.go` program. If it's not needed, it should be removed to avoid confusion.